### ISSUE TITLE
Fix the cli_parse call

### ISSUE THREAD

**Issue by Szasza** [domainaware/parsedmarc#495]
It looks like that the `cli_parse` call broke during one of the PR merges.
```
TypeError: cli_parse() takes 7 positional arguments but 8 were given 
```

**Comment by Szasza** [comment]
@seanthegeek 8.9.0 is broken due to this issue.

**Comment by seanthegeek** [comment]
I've included the fix in the 8.9.1 release. Thanks!

**Comment by Szasza** [comment]
Awesome, thank you as well!
